src/templates/en/2019/chapter.html

```diff
@@ -3,3 +3,7 @@
 {% block title %}
 Chapter {{ chapter }} | The Web Almanac by HTTP Archive
 {% endblock %}
+
+{% block main %}
```

Comment:
If we follow the current directory structure, this will need the templates to be duplicated across every language we support - which I don't think is necessarily a good idea. I think we should have templates that are language agnostic, and content that is translated and put in the template separately. I have a suspicion that's part of what #70 is addressing?